Modelica/Electrical/Analog/Lines/M_OLine.mo

```diff
@@ -337,9 +336,9 @@ its current can be accessed for plotting.</p>
 <p>In the picture of the segment can be seen, that a single segment is asymmetric. 
 Connecting such asymmetric segments in a series forces also an asymmetric multi line. 
 To get a symmetric model which is useful for coupling and which guaranties the same pin properties, 
```

Comment:
* Sometimes it is called `symmetric`, sometimes `symmetrical`. Could be made consistent,
* `To get ...` could be changed to `To design ...` as proposed in the other comment.